### ISSUE TITLE
[CI/CD] Pipe through logs from DUT when running YAML scripts

### ISSUE THREAD

**Issue by cecille** [project-chip/connectedhomeip#30847]
### Feature description

...
It would be good to get the DUT logs as well when running the YAML scripts in the CI. Right now, it's nearly impossible to debug anything.

### Platform

python

**Comment by drempelg** [comment]
yeah, been trying to figure out why a test is failing and being able to add debug logs while running in the exact environment the test is running in would be very helpful here (the test suite).

**Comment by cecille** [comment]
Need to check if this is still happening
